### PR TITLE
knative: Add unit tests for time and nullable utilities

### LINKED ISSUE #500: Add unit test coverage for Knative plugin utility helpers
Add unit test coverage for shared Knative plugin utilities (`getAge` and `isNullable`) to prevent regressions in time formatting and nullable field handling. Tests are deterministic and cover key edge cases without changing existing behavior.

### PR DESCRIPTION
**Pull request by intojhanurag**
## Summary

This PR adds unit test coverage for shared utility functions used by the Knative plugin:
- `getAge` (time formatting helper)
- `isNullable` (null/undefined guard helper)

## Fixed
#500 

These utilities are used across Knative views to format resource age and safely handle nullable API fields. Adding tests ensures deterministic behavior and guards against regressions during future refactors.

## Scope

- No functional or UI changes
- Tests only

### CODE REVIEW

**Comment by intojhanurag** [comment]
Hey @illume , Is this PR good to go ? Can you take  a look , If you have a moment :)

File: knative/src/utils/nullable.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { isNullable } from './nullable';
+
+describe('isNullable', () => {
+    it('should return true for null', () => {
+        expect(isNullable(null)).toBe(true);
+    });
+
+    it('should return true for undefined', () => {
+        expect(isNullable(undefined)).toBe(true);
+    });
```
[the remark this answers is not in the document]
Comment (illume) [reply knative/src/utils/nullable.test.ts]: Should be fixed with npm run format

File: knative/src/utils/time.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+describe('getAge', () => {
+    let dateNowSpy: any;
+    const MOCK_NOW = 1704067200000; // 2024-01-01T00:00:00.000Z
+
+    beforeAll(() => {
+        dateNowSpy = jest.spyOn(Date, 'now').mockReturnValue(MOCK_NOW);
+    });
+
+    afterAll(() => {
+        dateNowSpy.mockRestore();
```
[the remark this answers is not in the document]
Comment (illume) [reply knative/src/utils/time.test.ts]: Please use vitest ones?
